docs/es/week12/12-2.md

```diff
@@ -0,0 +1,600 @@
+
+<!--Moreover, BERT and RoBERTa achieve superhuman performance on SQUAD and Glue. The textual summaries generated by BART look very real to humans (high BLEU scores). These facts are evidence that the models do understand language in some way.-->
+
+Además, BERT y RoBERTa logran un rendimiento sobrehumano en SQUAD y Glue. Los resúmenes textuales generados por BART parecen muy reales para los humanos (puntajes BLEU altos). Estos hechos son evidencia de que los modelos entienden el lenguaje de alguna manera.
+
+<!--#### Grounded Language-->
+
+#### Lenguaje fundamentado
+
+<!--Interestingly, the lecturer (Mike Lewis, Research Scientist, FAIR) is working on a concept called ‘Grounded Language’. The aim of that field of research is to build conversational agents that are able to chit-chat or negotiate. Chit-chatting and negotiating are abstract tasks with unclear objectives as compared to text classification or text summarization.-->
+
+Curiosamente, el profesor (Mike Lewis, investigador científico, FAIR) está trabajando en un concepto llamado "Lenguaje fundamentado". El objetivo de ese campo de investigación es construir agentes conversacionales que sean capaces de charlar o negociar. Charlar y negociar son tareas abstractas con objetivos poco claros en comparación con la clasificación o el resumen del texto.
+
+<!--#### Can we evaluate whether the model already has world knowledge?-->
+
+#### ¿Podemos evaluar si el modelo ya tiene conocimiento global?
+
+<!--‘World Knowledge’ is an abstract concept. We can test models, at the very basic level, for their world knowledge by asking them simple questions about the concepts we are interested in.  Models like BERT, RoBERTa and T5 have billions of parameters. Considering these models are trained on a huge corpus of informational text like Wikipedia, they would have memorized facts using their parameters and would be able to answer our questions. Additionally, we can also think of conducting the same knowledge test before and after fine-tuning a model on some task. This would give us a sense of how much information the model has ‘forgotten’.-->
+
+El "conocimiento mundial" es un concepto abstracto. Podemos probar modelos, en un nivel muy básico, para su conocimiento del mundo haciéndoles preguntas simples sobre los conceptos que nos interesan. Modelos como BERT, RoBERTa y T5 tienen miles de millones de parámetros. Teniendo en cuenta que estos modelos están entrenados en un enorme corpus de texto informativo como Wikipedia, habrían memorizado hechos usando sus parámetros y podrían responder nuestras preguntas. Además, también podemos pensar en realizar la misma prueba de conocimientos antes y después de ajustar un modelo en alguna tarea. Esto nos daría una idea de cuánta información ha "olvidado" el modelo.
```

Comment:
May I suggest rephrasing "Podemos probar modelos... para su conocimiento del mundo..." as "Podemos probar el conocimiento del mundo que tienen los modelos (en un nivel muy básico)..."?

Reply:
Great suggestions overall, and also great bug-detecting! implemented all of them, thanx!